infrastructure/cli/initializer.go

```diff
@@ -124,6 +137,8 @@ func (i *Initializer) installCli() {
 
 	if cliPath != "" {
 		i.notifier.Send(lsp.SnykIsAvailableCli{CliPath: cliPath})
+		version := i.versionSource.CliVersion()
+		log.Info().Msg("snyk-cli: " + version + " (" + cliPath + ")")
```

Comment:
Suggestion: Currently the CLI info is only printed after being update/install. It would be helpful if the CLI info would also be part of the logs when not being updated.